cypress/integration/view_warning_note_details_adult.ts

```diff
@@ -0,0 +1,34 @@
+describe('Adult Group', () => {
+  beforeEach(() => {
+    cy.getCookies().should('be.empty');
+    cy.setCookie('hackneyToken', Cypress.env('TEST_KEY_ADULT_GROUP'));
+    cy.getCookie('hackneyToken').should(
+      'have.property',
+      'value',
+      Cypress.env('TEST_KEY_ADULT_GROUP')
+    );
+    cy.visit(
```

Comment:
You can replace this with the new `cy.visitAs` method

Reply:
Done in b98d427 🎉